acceptance/tests/invalid_ssl_config.rb

```diff
@@ -21,7 +21,7 @@
 invalid_config_mismatching_keys = {:broker_ws_uri => broker_ws_uri(master),
                                    :ssl_key => ssl_key_file(agent1, 1),
                                    :ssl_ca_cert => ssl_ca_file(agent1),
-                                   :ssl_cert => alt_ssl_cert_file(agent1, 1)}
+                                   :ssl_cert => ssl_cert_file(agent1, 1, use_alt_path=true)}
```

Comment:
@james-stocks also raised #265 as a non-ruby 2.0 requiring version to correct this.